spec/features/signing_up_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+# User visits sign up route
+# User enters email + password
+# clicks submit
+# Sees "you have signed up
+RSpec.feature "signing up" do
+  scenario "signs up" do
+    visit new_user_registration_path
+    fill_in 'Email', with: 'example@example.com'
+    fill_in 'Password', with: 'jep_password'
+    fill_in 'Password confirmation', with: 'jep_password'
+    click_button "Sign up"
+    expect(page).to have_content("Hello, example@example.com")
+  end
+
+  scenario "user leaves email blank" do
+    visit new_user_registration_path
+    fill_in :user_email, with: ''
```

Comment:
You should use the field label here, as you've done with the first test in this file.